jormungandr/src/rest/v1/logic.rs

```diff
@@ -91,11 +93,20 @@ pub async fn post_fragments(
         .map(|fragment| fragment.id().to_string())
         .collect();
     let mut msgbox = context.try_full()?.transaction_task.clone();
-    for fragment in fragments.into_iter() {
-        let msg = TransactionMsg::SendTransaction(FragmentOrigin::Rest, vec![fragment]);
-        msgbox.try_send(msg)?;
+    let (reply_handle, reply_future) = intercom::unary_reply();
+    let msg = TransactionMsg::SendTransactions {
+        origin: FragmentOrigin::Rest,
+        fragments,
+        fail_fast: true,
+        reply_handle,
+    };
+    msgbox.try_send(msg)?;
+    let reply = reply_future.await?;
+    if reply.is_error() {
+        Err(Error::Fragments(fragment_ids))
+    } else {
+        Ok(fragment_ids)
```

Comment:
It looks like it does not change the response in `Ok` case...

Reply:
See https://github.com/input-output-hk/jormungandr/pull/3301#discussion_r635829394

Reply:
Hm? It should not.